cyhy_report/customer/graphs.py

```diff
@@ -187,7 +188,7 @@ def plot(self, filename, size=1.0):
                     width > 0
                 ):  # TODO handle too labels getting squeezed, need box width in points
```

Comment:
Unrelated to your changes, but is this supposed to be "two"?

Reply:
I have no idea, but I think you may be right.  That is a @felddy comment from way back in the day.

Reply:
I don't see how it could be anything but a typo.  I'd go ahead and change it to "two" to avoid future confusion, but if you want to wait for @felddy to chime in that's OK too.

Reply:
Fair enough- changed in https://github.com/cisagov/cyhy-reports/pull/54/commits/a40a6a962c91731b52c8eaf40136eb7cdc093d47.